DirectX11/IniHandler.cpp

```diff
@@ -1218,6 +1221,63 @@ static void EnumerateCustomShaderSections(IniSections &sections, wchar_t *iniFil
 		customShaders[shader_id];
 	}
 }
+
+static void ParseSamplerState(CustomShader *shader, const wchar_t *section, wchar_t *iniFile)
+{
+	D3D11_SAMPLER_DESC* desc = &shader->sampler_desc;
+	wchar_t setting[MAX_PATH];
+	wchar_t key[32];
+	int i;
+	bool found;
+
+	memset(desc, 0, sizeof(D3D11_SAMPLER_DESC));
+
+	//TODO: do not really understand the difference between normal and comparison filter 
```

Comment:
Can't help here.  I'm a good code guy, but I don't understand a lot of the nuances of 3D and graphic programming.

Might be worth waiting for some feedback once people start to use and make suggestions.  Other people may have better ideas for best way to do sampling or ways to improve the image.